web/main.go

```diff
@@ -11,8 +11,11 @@ import (
 
 	"github.com/linkerd/linkerd2/controller/api/public"
 	"github.com/linkerd/linkerd2/pkg/admin"
+	"github.com/linkerd/linkerd2/pkg/config"
 	"github.com/linkerd/linkerd2/pkg/flags"
 	"github.com/linkerd/linkerd2/web/srv"
+
```

Comment:
There shouldn't be a space between imports here, and the imports should be sorted alphabetically. This should fix it:

```diff
diff --git a/web/main.go b/web/main.go
index f4338c4d..9b252d3e 100644
--- a/web/main.go
+++ b/web/main.go
@@ -13,9 +13,8 @@ import (
 	"github.com/linkerd/linkerd2/pkg/admin"
 	"github.com/linkerd/linkerd2/pkg/config"
 	"github.com/linkerd/linkerd2/pkg/flags"
-	"github.com/linkerd/linkerd2/web/srv"
-
 	pkgK8s "github.com/linkerd/linkerd2/pkg/k8s"
+	"github.com/linkerd/linkerd2/web/srv"
 	log "github.com/sirupsen/logrus"
 )
 
```